common-proto/src/main/proto/systems/ajax/malov/commonproto/stock/short_stock_response.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+import "systems/ajax/malov/commonproto/stock/big_decimal.proto";
+
+package systems.ajax.malov.commonproto.stock;
+
+option java_multiple_files = true;
+
```

Comment:
All files should be ordered in the following manner:

License header (if applicable)
File overview
Syntax
Package
Imports (sorted)
File options
Everything else